arch/xtensa/src/common/xtensa_hostfs.c

```diff
@@ -0,0 +1,231 @@
+/****************************************************************************
+ * arch/xtensa/src/common/up_hostfs.c
+ *
+ *   Copyright (C) 2018 Pinecone Inc. All rights reserved.
+ *   Author: Xiang Xiao <xiaoxiang@pinecone.net>
```

Comment:
@xiaoxiang781216 
is this you?
this file is based on arch/arm/src/common/up_hostfs.c.
may i change this license to apache 2.0?

Reply:
Sure.

Reply:
thank you. done.